otherlibs/unix/unixsupport.h

```diff
@@ -61,19 +61,28 @@ struct filedescr {
 #define Handle_val(v) (((struct filedescr *) Data_custom_val(v))->fd.handle)
 #define Socket_val(v) (((struct filedescr *) Data_custom_val(v))->fd.socket)
 #define Descr_kind_val(v) (((struct filedescr *) Data_custom_val(v))->kind)
-#define CRT_fd_val(v) (((struct filedescr *) Data_custom_val(v))->crt_fd)
```

Comment:
We could still keep the `#define` but as an alias for the getter function:
```
#define CRT_fd_val caml_win32_get_CRT_fd
```
There's a small chance that 3rd-party code uses `CRT_fd_val` for reading the `crt_fd` field, and with this addition it would still work.  3rd-party code that changes the `crt_fd` field using `CRT_fd_val` should not do that.

Reply:
Done in the last commit.